deploy/gke-marketplace-app/README.md

```diff
@@ -60,15 +60,16 @@ Currently, GKE >= 1.18.7 only supported in GKE rapid channel, to find the latest
 export PROJECT_ID=<your GCP project ID>
 export ZONE=<GCP zone of your choice>
 export REGION=<GCP region of your choice>
-export DEPLOYMENT_NAME=<GKE cluster name, triton_gke for example>
+export DEPLOYMENT_NAME=<GKE cluster name, triton-gke for example>
 
 gcloud beta container clusters create ${DEPLOYMENT_NAME} \
 --addons=HorizontalPodAutoscaling,HttpLoadBalancing,Istio \
 --machine-type=n1-standard-8 \
 --node-locations=${ZONE} \
 --subnetwork=default \
 --scopes cloud-platform \
---num-nodes 1
+--num-nodes 1 \
+--project ${PROJECT_ID}
 
 # add GPU node pools, user can modify number of node based on workloads
```

Comment:
@mengdong Creating a cluster and adding GPU nodes could take up-to 10 minutes. Please be patient after executing this command.

Reply:
added